src/data/List.ts

```diff
@@ -2160,6 +2161,10 @@ class List<
             }
         };
 
+        validateDimension = function (list: List, dim: DimensionName): boolean {
```

Comment:
I think use a static function does not match the convention of this file.
And as the param should better in type `DimensionLoose` rather than only `DimensionName`.

I think it probably better to make a function like:

```ts
class List {
    hasDimension(dim: DimensionLoose): boolean {
        return !!this.getDimensionInfo(dim);
    }
}
```